internal/build/buildkit_printer.go

```diff
@@ -8,6 +8,8 @@ import (
 
 	digest "github.com/opencontainers/go-digest"
 
+	"github.com/tonistiigi/units"
+
```

Comment:
nit: should there be spaces between all these packages?

Reply:
done!